2-mandatory/2-financial-times.js

```diff
@@ -5,7 +5,13 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
```

Comment:
Almost perfect - but same comment as above about using `let` or `const` 😄 
For an extra challenge, try re-writing the below function using the `filter` array method.